saleor/graphql/order/types.py

```diff
@@ -265,7 +266,12 @@ def resolve_total_balance(self, info):
 
     @staticmethod
     def resolve_fulfillments(self, info):
-        return self.fulfillments.all().order_by('pk')
+        context_kwargs = getattr(info.context, 'kwargs', {})
```

Comment:
I'm not 100% convinced about assigning arbitrary variables to `info.context` which in Graphene is a Django request object, but honestly, I also don't see a better idea how to customize the resolver behavior for `me` query.

What about a different approach - would it make sense if we excluded canceled fulfillments for unauthenticated users (customers)?

Reply:
I feel the same that it is not the best approach. It will nice if graphene supports passing context to next levels natively.

Reply:
The ```me``` API requires authentication, and ```me->orders->fulfillments``` which is used by frontend should exclude cancelled orders.

So I think it won't achieve this result if we only exclude cancelled fulfillments for unauthenticated users?

Reply:
Sorry, my bad- I was thinking about users who are not staff, so regular customers.

Reply:
This sounds quite reasonable :+1: